test/csuite/CMakeLists.txt

```diff
@@ -195,6 +195,12 @@ define_c_test(
     DEPENDS "WT_POSIX"
 )
 
+define_c_test(
```

Comment:
This should be added after line 298 to keep the order correct.

Reply:
Oops, I forgot to move it after renamed the test. Good catch, thanks!